OpenSim/Common/APDMDataReader.cpp

```diff
@@ -67,17 +67,17 @@ APDMDataReader::extendRead(const std::string& fileName) const {
     bool newFormat = false;
     if (tokens[0] == "Format=7") {
         newFormat = true;
-        dataRate = 128; // Will fix on reading first 2 rows
+        dataRate = 128; // Will fix after reading computing it from time column
```

Comment:
Quick question @aymanhab does APDM not report data rate because they do not guarantee a fixed sampling rate? Given that each sample is time stamped, it is possible that the sampling rate is not perfectly uniform. In that case, reporting a data rate based on the first two samples won't be right.

Reply:
The two APDM formats handle this differently, in the old one they reported dataRate and we assumed it's fixed by sensor and calculated time, the later format has no reference to data rate and the data is timestamped. I'm leaning to remove the dataRate reference altogether and just report the times relative to starttime with no units attached. Can't find documentation anywhere, I'm in touch with their person to try to get more info.